protocol/message_persistence.go

```diff
@@ -2967,9 +2979,26 @@ func (db sqlitePersistence) findStatusMessageIdsReplies(tx *sql.Tx, bridgeMessag
 	return statusMessageIDs, nil
 }
 
-// Finds status messages id which are replies for bridgeMessageID
-func (db sqlitePersistence) findStatusMessageIdsRepliedTo(tx *sql.Tx, parentMessageID string) (string, error) {
-	rows, err := tx.Query(`SELECT user_messages_id FROM bridge_messages WHERE message_id = ?`, parentMessageID)
+func (db sqlitePersistence) FindStatusMessageIdForBridgeMessageId(messageID string) (string, error) {
```

Comment:
Does this mean it could get slower over time? Excuse my ignorance, are the messages split per chats or something?

Reply:
Here, we query from bridge messages.
Bridge messages have a separated table. They are joined with messages via a messageId.
I did not investigate how much time does it take.